roles/v2v.ci/tasks/miq_set_provider_concurrent_vm_migration_max.yml

```diff
@@ -0,0 +1,44 @@
+---
+- block:
+  - name: Get RHV provider ID
+    uri:
+      url: "https://{{ inventory_hostname }}/api/providers/?filter[]=name={{ v2v_ci_rhv_name }}&expand=resources"
+      user: "{{ miq_app_username | default('admin') }}"
+      password: "{{ miq_app_password | default('smartvm') }}"
+      method: GET
+      validate_certs: no
+    register: rhv_provider_json
+
+  - set_fact:
+      query: "resources[?name=='{{ v2v_ci_rhv_name }}'].id"
+
+  - name: Extracting RHV provider id from JSON
+    set_fact:
+      rhv_provider_id: "{{ rhv_provider_json.json | json_query(query) | first }}"
+
+  - name: Get custom attribute {{ provider_concurrent_max_name }}
+    uri:
+      url: "https://{{ inventory_hostname }}/api/providers/{{ rhv_provider_id }}/custom_attributes/?filter[]=name={{ provider_concurrent_max_name | regex_replace(' ', '%20') }}"
+      user: "{{ miq_app_username }}"
+      password: "{{ miq_app_password }}"
+      method: GET
+      validate_certs: no
+    register: max_transorm_runners_json
+
+  - name: Add custom attribute {{ provider_concurrent_max_name }}
+    uri:
+      url: "https://{{ inventory_hostname }}/api/providers/{{ rhv_provider_id }}/custom_attributes/"
+      user: "{{ miq_app_username | default('admin') }}"
+      password: "{{ miq_app_password | default('smartvm') }}"
```

Comment:
please change the default password to 123456

Reply:
This is the default password of CFME appliance.